packages/devtools-modules/index.js

```diff
@@ -2,12 +2,20 @@ const Services = require("./client/shared/shim/Services");
 const SplitBox = require("./client/shared/components/splitter/SplitBox");
 // const SplitBoxCSS = require("./client/shared/components/splitter/SplitBox.css")
 const sprintf = require("./shared/sprintf").sprintf;
+const Tabbar = require("./client/shared/components/tabs/tabbar");
+const TabPanel = require("./client/shared/components/tabs/tabs").TabPanel;
+const TreeRow = require("./client/shared/components/tree/tree-row");
+const TreeView = require("./client/shared/components/tree/tree-view");
```

Comment:
Reverted in 4482a0b. Sorry for the revert, but this was breaking build steps in other packages.